src/cascadia/TerminalApp/Tab.idl

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace TerminalApp
+{
+    [default_interface] runtimeclass Tab : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    {
+        String Title;

```

Comment:
These might benefit from being `String Title { get; };` -- if they only have getters, you don't need to implement setters.

Unless that's required by the property notifier.

Reply:
(but I don't think that they should be inherently settable by external users)